microcosm_pubsub/envelope.py

```diff
@@ -133,7 +133,7 @@ def parse_raw_message(self, consumer, raw_message):
         message_id = self.parse_message_id(raw_message)
         receipt_handle = self.parse_receipt_handle(raw_message)
         attributes = raw_message.get("Attributes", {})
-        approximate_receive_count = attributes.get("ApproximateReceiveCount")
+        approximate_receive_count = int(attributes.get("ApproximateReceiveCount", 1))
```

Comment:
I'm not sure if this is wholly correct here, but it will allow a fallback to a 1 second timeout if not provided.